packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json

```
@@ -1,7 +1,10 @@
 {
   "dedicatedCloud_datacenter_backup_new_description1": "Il semblerait que vous n’avez pas activé de service de sauvegarde",
   "dedicatedCloud_datacenter_backup_new_description2": "Vous pouvez choisir entre 3 nouvelles offres pour la sauvegarde de vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
+  "dedicatedCloud_datacenter_backup_new_description3": "Cette offre sera disponible par défaut pour tous les datacentres de ce private cloud.",
```

```suggestion
  "dedicatedCloud_datacenter_backup_new_description3": "Cette offre sera disponible par défaut pour tous les datacentres de ce Private Cloud.",
```
Same for below sentences

---

This has been changed